api-2.0/src/test/java/org/openmrs/module/fhir/api/RelatedPersonServiceTest.java

```diff
@@ -20,17 +21,24 @@
 
 	private static final String LOC_INITIAL_DATA_XML = "RelatedPersonServiceTest_additionalTestData.xml";
 
+	protected static final String PAT_SEARCH_DATA_XML = "org/openmrs/api/include/PatientServiceTest-findPatients.xml";
```

Comment:
Do you have any reason for not making this private, instead of protected?

Reply:
No reason at all. will change it to private